frontend/src/layouts/ContentLayout.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { CssBaseline, Paper } from '@mui/material';
+
+const styles = {
+  main: {
+    width: 'auto',
+    display: 'block', // Fix IE 11 issue.
+    marginLeft: 100,
```

Comment:
I think removing these `margin`s is the way to go after the changes. As it squeezes the content in low resolution screens, while not doing anything on medium+ resolutions.

### With margin:

#### Low Resolution
<img src="https://user-images.githubusercontent.com/26660856/202904172-d5bf88aa-cc35-456c-a93a-aa580807a4b2.png" width=375>

#### High Resolution
![resim](https://user-images.githubusercontent.com/26660856/202904399-663a45ad-cbb7-45b1-ae51-5f7b74d077de.png)


### Margins removed: 

#### Low Resolution
<img src="https://user-images.githubusercontent.com/26660856/202904203-8547d841-1420-4511-84d3-25ce2d54d853.png" width=375>

#### High resolution
![resim](https://user-images.githubusercontent.com/26660856/202904318-ea227661-13a4-4834-8e6d-0b03b20308a6.png)


My alternative would be setting width to another value than '100%' to leave space or increase padding on main container.

Reply:
This was fixed in commit ae7e0e1